docs/specification.md

```diff
@@ -634,27 +629,33 @@ type VoteType = Upvote | Downvote
 data VoteParam = VoteParam
   { proposal_id :: ProposalId
   , vote_type :: VoteType
+  , vote_amount :: Natural
```

Comment:
This can be dangerous since it is probably not obvious for the user. The best choice would be to have a separate `vote_all` entrypoint (which accept only 1 `VoteParams`, `vote` currently accepts list of `VoteParam`). But this is just my opinion, if you are good with that, I'm good as well, Ivan.

Reply:
Okay, let's ignore this suggestion for now, it's just a suggestion, not a requirement. Things are simpler without it.